packages/build-tools/src/builders/ios.ts

```diff
@@ -116,14 +99,28 @@ async function buildAsync(ctx: BuildContext<Ios.Job>): Promise<string> {
     await ctx.cacheManager?.saveCache(ctx);
   });
 
-  return await ctx.runBuildPhase(BuildPhase.UPLOAD_ARTIFACTS, async () => {
-    const buildArtifacts = await findBuildArtifacts(
+  return await ctx.runBuildPhase(BuildPhase.UPLOAD_APPLICATION_ARCHIVE, async () => {
+    try {
+      const xcodeBuildLogsPath = await findXcodeBuildLogsPath(ctx);
+      if (xcodeBuildLogsPath) {
+        await ctx.uploadArtifacts(ArtifactType.XCODE_BUILD_LOGS, [xcodeBuildLogsPath], ctx.logger);
+      }
+    } catch {
```

Comment:
`findXcodeBuildLogsPath` should never throw and if `uploadArtifacts` is throwing an error then it's likely because of some network issue and marking this build as successful might not be correct behavior.

I'm not saying that this approach is wrong, but it's not clear what this try-catch is supposed to catch and for what reason.